### ISSUE TITLE
On Crimes and Disease

### ISSUE THREAD

**Issue by JFDaly** [LoneGazebo/Community-Patch-DLL#1139]
As discussed, the changes to how the AI interprets the worth of Buildings may need some exceptions in terms of my Crime and Disease yields, although my understanding of this change is still rudimentary, so I'll explain how Crime and Disease are implemented in the hopes that you'll be able to ascertain the best way of accomodating them.

Crime is produced by many buildings and Specialists, in place of Maintenance, and is generally a bad thing, but implemented as a positive yield (i.e. through Building_YieldChanges). From my understanding, this would make the AI favour buildings which produce Crime. Because Crime replaces Maintenance, however, I'm not sure an exception is needed here. But, there are certain Buildings (Jail, Constabulary, and Police Station) that reduce Crime via modifier (i.e. Building_YieldModifiers), which naturally uses a negative modifier. For this, I think an exception would be needed, as otherwise the AI would devalue these buildings when they should consider them more valueable. There is also a custom table, Building_JFD_CrimeMods, which reduces specific Crimes (as Crimes are broken down according to yield distributions, e.g. Food == Violence, Faith == Heresy, etc.), which would make certain buildings more valuable. There's probably no real way to factor the value of this in, however.

Disease is similarly produced by Building_YieldChanges (when on buildings) and is similarly negative in actualization, and similarly Building_YieldModifiers with a negative value would actually be a beneficial thing. But in this case, I intend to use negative flavours in order to indicate to the AI that Disease, predominantly, affects Growth. But the same exception would therefore be needed here for Building_YieldModifiers.

**TL;DR** Building/Policy_YieldModifiers for Crime (YIELD_JFD_CRIME, if needed) and Disease (YIELD_JFD_DISEASE) should be considered positive when negative and negative when positive. Or, at least, completely ignored by the AI and left to the flavours.

**Comment by LoneGazebo** [comment]
Done. AI now has a rudimentary understanding of these values.
